### PR TITLE
API Gateway 1.8.0

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Align CRD with API Gateway
- Allow "http" issuer and jwks
- Update API Gateway image to 1.8.0

### CODE REVIEW

**Review by triffer — APPROVED** [review]

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 1]
```diff
@@ -246,37 +252,40 @@ spec:
...
-                            description: Configures the handler. Configuration
-                              keys vary per handler.
+                            description: Configures the handler. Configuration keys
+                              vary per handler.
                             type: object
                             x-kubernetes-preserve-unknown-fields: true
                             properties:
                               jwks_urls:
                                 description: Specifies the array of URLs from which Ory Oathkeeper can retrieve JSON Web Keys for validating JSON Web Token.
+                                  Must start with either "http://", "https://" or "file://".
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:276]:
```suggestion
                                  The value must begin with either `http://`, `https://`, or `file://`.
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 2]
```diff
@@ -339,6 +352,14 @@ spec:
                       - name
                       - port
                       type: object
+                    timeout:
+                      description: Specifies the timeout, in seconds, for
+                        HTTP requests made to spec.rules.path. The maximum timeout is limited
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:357]:
```suggestion
                        HTTP requests made to **spec.rules.path**. The maximum timeout is limited
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 3]
```diff
@@ -54,38 +55,40 @@ spec:
...
                                 type: array
                                 items:
                                   type: string
-                                  pattern: ^(https://|file://).*$
+                                  pattern: ^(http://|https://|file://).*$
                               trusted_issuers:
-                                description: If the **trusted_issuers** field is set, the JWT must contain a value for the claim `iss` 
+                                description: If the **trusted_issuers** field is set, the JWT must contain a value for the claim `iss`
                                   that matches exactly (case-sensitive) one of the values of **trusted_issuers**.
+                                  Must start with either "http://", "https://" or "file://".
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:87]:
```suggestion
                                  The value must begin with either `http://`, `https://`, or `file://`.
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 4]
```diff
@@ -286,21 +295,23 @@ spec:
...
+                      description: Represents the list of allowed HTTP request methods
+                        available for the **spec.rules.path**.
                       items:
                         type: string
                       minItems: 1
                       type: array
                     mutators:
-                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator) mutators.
+                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator)
+                        mutators.
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:306]:
```suggestion
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 5]
```diff
@@ -286,21 +295,23 @@ spec:
...
-                      description: Represents the list of allowed HTTP request methods available for the **spec.rules.path**.
+                      description: Represents the list of allowed HTTP request methods
+                        available for the **spec.rules.path**.
                       items:
                         type: string
                       minItems: 1
                       type: array
                     mutators:
-                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator) mutators.
+                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator)
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:305]:
```suggestion
                      description: Specifies the list of [Ory Oathkeeper mutators](https://www.ory.sh/docs/oathkeeper/pipeline/mutator).
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 6]
```diff
@@ -246,37 +252,40 @@ spec:
...
+                                  The value must begin with either `http://`, `https://`, or `file://`.
                                 type: array
                                 items:
                                   type: string
-                                  pattern: ^(https://|file://).*$
+                                  pattern: ^(http://|https://|file://).*$
                               trusted_issuers:
                                 description: If the **trusted_issuers** field is set, the JWT must contain a value for the claim `iss` 
                                   that matches exactly (case-sensitive) one of the values of **trusted_issuers**.
+                                  Must start with either "http://", "https://" or "file://".
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:284]:
```suggestion
                                  The value must begin with either `http://`, `https://`, or `file://`.
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 7]
```diff
@@ -54,38 +55,40 @@ spec:
...
+                            description: Configures the handler. Configuration keys
+                              vary per handler.
                             type: object
                             x-kubernetes-preserve-unknown-fields: true
                             properties:
                               jwks_urls:
-                                description: Specifies the array of URLs from which Ory Oathkeeper can retrieve 
-                                  JSON Web Keys for validating JSON Web Token.
+                                description: Specifies the array of URLs from which Ory Oathkeeper can retrieve JSON Web Keys for validating JSON Web Token.
+                                  Must start with either "http://", "https://" or "file://".
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:79]:
```suggestion
                                  The value must begin with either `http://`, `https://`, or `file://`.
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 8]
```diff
@@ -95,21 +98,23 @@ spec:
...
+                      description: Represents the list of allowed HTTP request methods
+                        available for the **spec.rules.path**.
                       items:
                         type: string
                       minItems: 1
                       type: array
                     mutators:
-                      description: Specifies the list of [Oathkeeper mutators](https://www.ory.sh/docs/oathkeeper/pipeline/mutator).
+                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator)
+                        mutators.
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:109]:
```suggestion
```

File: installation/resources/crds/api-gateway/apirules.gateway.crd.yaml [hunk 9]
```diff
@@ -95,21 +98,23 @@ spec:
...
-                      description: Represents the list of allowed HTTP request methods available for the **spec.rules.path**.
+                      description: Represents the list of allowed HTTP request methods
+                        available for the **spec.rules.path**.
                       items:
                         type: string
                       minItems: 1
                       type: array
                     mutators:
-                      description: Specifies the list of [Oathkeeper mutators](https://www.ory.sh/docs/oathkeeper/pipeline/mutator).
+                      description: Specifies the list of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/mutator)
```
Comment (nataliasitko) [inline installation/resources/crds/api-gateway/apirules.gateway.crd.yaml:108]:
```suggestion
                      description: Specifies the list of [Ory Oathkeeper mutators](https://www.ory.sh/docs/oathkeeper/pipeline/mutator).
```

**Review by nataliasitko — COMMENTED** [review]
I'm sorry, I missed those. These are the last suggestions, I promise

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**